### DISCUSSION TITLE
Can I embed cards on a website?

### DISCUSSION THREAD

**Discussion by snowy-jaguar** [anuraghazra/github-readme-stats#1677] [Q&A] [answered]
Hi, 

I have a website built on Wix and was wondering if it was possible for me to embed these stat cards on it (the language stat card mostly but maybe a few of the others).

Thanks for any information/advice/help that anyone is able to provide.

**Comment by anuraghazra** [comment, marked as answer]
Yes you can, just use an <img \/> tag. 

```html
<img src="README_STATS_URL" alt="My readme stats" />
```

**Reply by snowy-jaguar** [reply]
Thanks

**Reply by snowy-jaguar** [reply]
> Yes you can, just use an <img /> tag.
> 
> ```
> <img src="README_STATS_URL" alt="My readme stats" />
> ```

Is there a way to edit the wdith of the stat card
<img width="358" alt="image" src="https://user-images.githubusercontent.com/51423344/159918897-666469b9-732f-404f-a1d0-81d1c2ed29ae.png">

**Reply by rickstaa** [reply]
@SnowyJaguar1034 Thanks for your question. I noticed that you also asked this question in #1748. I will therefore post my reply in two places.

There are multiple ways to achieve the desired behaviour. The first way is by using the HTML `width` and `height` attributes. This is what I use in my readme. For more information on the available attributes see https://www.w3schools.com/tags/tag_img.asp:

```html
<img height=200 width=200 src="https://github-readme-stats-git-master-rstaa-rickstaa.vercel.app/api/top-langs/?username=rickstaa&layout=compact&langs_count=10&hide_border=1&role=OWNER,COLLABORATOR&theme=dark#gh-dark-mode-only" alt="Rick Staa's Language stats" />
</a>
```

<img height=200 width=200 src="https://github-readme-stats.vercel.app/api?username=SnowyJaguar1034" alt="SnowyJaguar1034's Github Stats" />
</a>

We also have #1334 open to allow users to change the width of the card directly in the SVG. I, however, did not have time yet to look at this PR, but you can already use it in [your vercel instance](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own-vercel-instance).
